constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/Motion.java

```diff
@@ -1748,4 +1761,27 @@ public boolean setValue(int id, boolean value) {
     public int getId(String name) {
         return 0;
     }
+    /**
+     * Set stagger scale
+     * @param staggerScale
+     */
+    public void setStaggerScale(float staggerScale) {
+        mStaggerScale = staggerScale;
+    }
+
+    /**
+     * set the offset used in calculating stagger launches
+     * @param staggerOffset
+     */
+    public void setStaggerOffset(float staggerOffset) {
+        mStaggerOffset = staggerOffset;
+    }
+
+    /**
+     * Get the motionStagger valule if they are not using positions
+     * @return
+     */
+    public float getMotionStagger() {
+        return mMotionStagger;
```

Comment:
It seems that we return the stagger value regardless of positions, is the javadoc wrong or the implementation?